dime-coding-standards/checklists/tex/hfcs_checklist.tex

```diff
@@ -0,0 +1,282 @@
+                
+                \item[] \dimeCheckBox{Bar graph for reasons for declining (categorical)}
+                
+            \end{itemize}
+
+
+            \item[]\dimeCheckBox{\textbf{Duration -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Bar graph of average duration for each day}
+                
+                \item[] \dimeCheckBox{Distribution of duration (either as k-desnity curves, or as box-plots by modules/sections}
+                
+            \end{itemize}
+
+            \end{itemize}
+            
+
+\subsection*{Survey quality}
+Tracking \textbf{survey quality} involves checks for the following:
+            \begin{itemize}
+            
+            \item[]\dimeCheckBox{\textbf{Duplicates -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Spreadsheet listing duplicate entries in ID and non-ID variables to share with the field team for corrections}
+                
+                
+            \end{itemize}
+            
+            \dimeCheckBox{\textbf{Distribution of key variables:}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[]\dimeCheckBox{Table with summary stats for all key variables, highlighting missing values and outliers} 
+                
+                \item[]\dimeCheckBox{Spreadsheet listing the errors in the data that need to be shared with the field team for corrections}
+                
+                \item[]\dimeCheckBox{Bar graphs/k-density curves for \textbf{continuous variables}}
+                
+                \item[]\dimeCheckBox{Graph for number of missing and extended missing responses (like \texttt{Do not know}, \texttt{Refuse to answer}) by question type }
+                
+                \item[]\dimeCheckBox{Graph for number of outliers by question type}
+                
+            \end{itemize}
+
+            \dimeCheckBox{\textbf{Inconsistency in responses -}} 
+            The high-frequency checks contain/export:
+            \begin{itemize}
+            
+                \item[]\dimeCheckBox{Spreadsheets listing inconsistent responses with questions and their values that needs to be shared with the field team for corrections. For example, inconsistencies across variables for the same respondent, or for the same variables across different time periods.}
+                
+            \end{itemize}
+
+            \dimeCheckBox{\textbf{Form version used -}} 
+            The high-frequency checks contain/export:
+            \begin{itemize}
+            
+                \item[]\dimeCheckBox{Table listing number of responses by form-version and date}
+                
+            \end{itemize}
```

Comment:
```suggestion
            \textbf{Form version used -}
            The high-frequency checks contain/export:
            \begin{itemize}
            
                \item[]\dimeCheckBox{Table listing number of responses by form-version and date}
                
            \end{itemize}
```
I am not sure when and how the check box in the upper level will be checked when the item in the list wont or vice versa. This could lead to confusion. I suggest removing the check boxes for the title item have check boxes only for `\items[]`.